frontend/src/features/directory/components/FilterableVendorTable.tsx

```diff
@@ -258,23 +256,11 @@ export function FilterableVendorTableContent({
             value={selectedLocation?.display_name || ""}
             onSelect={setSelectedLocation}
           />
-        </Box>
-
-        {/* Second Row: Other Filters */}
```

Comment:
can we keep the filters on a second row for now? I know they'll be moved later

Reply:
Done!